src/hotspot/cpu/aarch64/c1_LIRGenerator_aarch64.cpp

```diff
@@ -831,18 +831,12 @@ void LIRGenerator::do_LibmIntrinsic(Intrinsic* x) {
       }
       break;
     case vmIntrinsics::_dlog:
-      if (StubRoutines::dlog() != nullptr) {
-        __ call_runtime_leaf(StubRoutines::dlog(), getThreadTemp(), result_reg, cc->args());
-      } else {
-        __ call_runtime_leaf(CAST_FROM_FN_PTR(address, SharedRuntime::dlog), getThreadTemp(), result_reg, cc->args());
-      }
+      assert(StubRoutines::dlog() == nullptr, "no Math.log intrinsic on AArch64");
```

Comment:
Is there a reason to assert this? I think the comment should be enough. And it could be richer as well, something like: 

```
 // Math.log intrinsic is not implemented on AArch64 (see JDK-8210858),
 // but we can still call the shared runtime.
```